app/views/layouts/application.html.erb

```diff
@@ -10,7 +10,17 @@
   </head>
 
   <body>
+
+    <% if notice %>
```

Comment:
I reviewed with @adc17, change the notice if statement and you should be good to go